tests/linalg_test.py

```diff
@@ -1714,11 +1714,12 @@ def osp_fun(A):
     shape=[(3, 4), (3, 3), (4, 3)],
     dtype=[np.float32],
     mode=["full", "r", "economic"],
+    pivoting=[False, True]
   )
-  def testScipyQrModes(self, shape, dtype, mode):
+  def testScipyQrModes(self, shape, dtype, mode, pivoting):
```

Comment:
This probably needs to be skipped when `pivoting is True` unless the test is running on CPU. You can probably use something like [`if not jtu.test_device_matches(["cpu"])`](https://github.com/jax-ml/jax/blob/ac817b48ca90e08da382b1fe674fa58122b7dadd/jax/_src/test_util.py#L459C5-L459C24).

Reply:
Oh and also, this should be conditioned on the version of `jaxlib` too, since the currently released backend doesn't support this FFI call. You can use the [`jtu.jaxlib_version`](https://github.com/jax-ml/jax/blob/ac817b48ca90e08da382b1fe674fa58122b7dadd/jax/_src/test_util.py#L339C5-L339C19) utility for conditioning on that.